### ISSUE TITLE
Missing syntax error for trailing backslash at end of file

### ISSUE THREAD

**Issue by correctmost** [microsoft/pyright#10814]
**Describe the bug**

Pyright does not emit a syntax error when there's a trailing backslash at the end of a file.

**Code or Screenshots**

```python
\
```

The same file causes a syntax error when run:
```
$ python y.py 
  File "y.py", line 1
    \
SyntaxError: unexpected EOF while parsing
```

**VS Code extension or command-line**
Command-line version 1.1.403

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.405
